contexts/Web3Context.tsx

```diff
@@ -30,8 +30,8 @@ interface Web3ContextData {
 export const Web3Context = createContext<Web3ContextData>({} as Web3ContextData)
 
 // TODO: Move to env
-const DEFAULT_NETWORK = { NAME: 'bscTestnet', CHAIN_ID: 97 }
-// const DEFAULT_NETWORK = { NAME: 'bsc', CHAIN_ID: 56 }
+//const DEFAULT_NETWORK = { NAME: 'bscTestnet', CHAIN_ID: 97 }
+const DEFAULT_NETWORK = { NAME: 'bsc', CHAIN_ID: 56 }
```

Comment:
precisa descomentar a testnet e comentar a mainet pra gente poder usar na url develop.mousehaunt.com
```suggestion
const DEFAULT_NETWORK = { NAME: 'bscTestnet', CHAIN_ID: 97 }
//const DEFAULT_NETWORK = { NAME: 'bsc', CHAIN_ID: 56 }
```